### PR TITLE
badblocks, ddrescue, fdisk, fsck, ioping, smartctl, wipefs: change /dev/sda into /dev/sdX

### PR DESCRIPTION
**Pull request by Agno94**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/ddrescue.md [hunk 1]
```diff
@@ -9,4 +9,4 @@
 
 - Clone Disk A to Disk B, creating a log file:
 
-`sudo ddrescue --force --no-scrape {{/dev/sda}} {{/dev/sdb}} {{path/to/log.txt}}`
+`sudo ddrescue --force --no-scrape {{/dev/sdX}} {{/dev/sdb}} {{path/to/log.txt}}`
```
Comment (navarroaxel) [inline pages/linux/ddrescue.md:12]:
```suggestion
`sudo ddrescue --force --no-scrape {{/dev/sdX}} {{/dev/sdY}} {{path/to/log.txt}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Just 1 comment.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @Agno94!

**Comment by sbrl** [comment]
Looks like there's a conflict preventing the merge here @Agno94 - would you be able to assist here at all please?

**Comment by navarroaxel** [comment]
@Agno94 Looks ready for merge, right?

**Comment by Agno94** [comment]
Yes

**Comment by navarroaxel** [comment]
@Agno94 I invite you to collaborate sending a PR for add this rule to the <https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md> file.

**Pull request merged**